### PR TITLE
Added the cliniccard component

### PR DESCRIPTION
**Pull request by Shari-alt**
I styled the empty cliniccard component with CSS.
There is no functionality yet. 
Would love to get a review from you - thank you ☀️

### CODE REVIEW

**Review by FriederikeM — APPROVED** [review]
looks good, maybe increase the box shadow around the clinic cards a little. it is hard to tell the difference between the card and the white background.
And maybe some more commits ;)

File: src/App.js [hunk 1]
```diff
@@ -33,7 +44,12 @@ function App() {
             </Route>
           </Switch>
         </main>
-        <footer className="Footer"></footer>
+        <Switch>
+          <Route exact path="/"></Route>
+          <Route path="/*">
```
Comment (namirsab) [inline src/App.js]: To be more exact, you could give here an array of strings for every matching path:
https://reactrouter.com/web/api/Route/path-string-string

File: src/components/ClinicCard.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import "./ClinicCard.css";
+import Download from "../images/Download.jpeg";
+import { ReactComponent as Edit } from "../images/Edit.svg";
+import { ReactComponent as Safe } from "../images/Safe.svg";
+
+export default function ClinicCard({ name, place }) {
+  return (
+    <div className="ClinicCard">
+      <img className="ClinicPicture" src={Download} alt="Clinicpicture" />
+      <h3 className="ClinicName"> Alibiname{name} </h3>
```
Comment (namirsab) [inline src/components/ClinicCard.js]: I guess you don't want to keep `Alibi` and `Wonderland` here, do you?

File: src/components/ClinicCard.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+.Edit {
+  position: relative;
+  left: -75px;
+}
+
+.Safe {
+  position: relative;
+  right: -75px;
+}
```
Comment (namirsab) [inline src/components/ClinicCard.css]: I see you are relatively positioning the elements here, even though the parent element is a flexbox. I think you could achieve the same result just using flex, giving the `ClinicCardButtons` a width of 100% and then using flex to align the buttons. We could take a look together if you want.

Using absolute/relative positioning should be used carefully, and you can achieve this without it

File: src/components/Home.css [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
 
 .ButtonList {
   display: flex;
   flex-direction: column;
   justify-content: space-around;
-  gap: 15px;
+  gap: 10px;
   position: relative;
-  top: 170px;
+  top: 220px;
```
Comment (namirsab) [inline src/components/Home.css]: Same here, you can achieve the same results with flex without using relative positioning:

**Review by namirsab — CHANGES_REQUESTED** [review]
hey @Shari-alt 
lovely!
I love your design and I think you did a great job!
I have some requests, mostly about positioning, please take a look!

File: src/components/Home.css (hunk 4 above)
Comment (namirsab) [reply src/components/Home.css]: image

**Pull request merged**